code/game/data_huds.dm

```diff
@@ -487,6 +490,14 @@
 	else
 		holder.icon_state = ""
 
+/*~~~~~~~~~~~~~~
+	JANI HUD
+~~~~~~~~~~~~~~~*/
+/obj/effect/decal/cleanable/proc/jani_hud_set_sign()
+	var/image/holder = hud_list[JANI_HUD]
+	holder.icon_state = "caution"
+	holder.plane = ABOVE_LIGHTING_PLANE
+
```

Comment:
Can you center the icon?

Reply:
Are you sure about that? It would block vision of what the cleanable is in some cases.

Reply:
You can add `mouse_opacity = MOUSE_OPACITY_TRANSPARENT` and icon size, shape and location is irrelevant.

Reply:
![image](https://github.com/ParadiseSS13/Paradise/assets/77684085/fa604225-6e4c-43e1-8b6e-2578c5816213)
Something like this?

Reply:
Yeah, this looks far better! Looking forward to the updated Icon!